tests/integration/flows/test_bigquery_to_adls.py

```diff
@@ -1,23 +1,70 @@
 from viadot.flows import BigQueryToADLS
 from prefect.tasks.secrets import PrefectSecret
+import pendulum
+import os
+
+ADLS_PATH = "raw/tests"
+FILE_NAME = str(pendulum.now("utc")) + ".parquet"
+BIGQ_CREDENTIAL_KEY = "BIGQUERY_TESTS"
+ADLS_CREDENTIAL_SECRET = PrefectSecret(
+    "AZURE_DEFAULT_ADLS_SERVICE_PRINCIPAL_SECRET"
+).run()
 
 
 def test_bigquery_to_adls():
-    credentials_secret = PrefectSecret(
-        "AZURE_DEFAULT_ADLS_SERVICE_PRINCIPAL_SECRET"
-    ).run()
+    flow_bigquery = BigQueryToADLS(
+        name="Test BigQuery to ADLS extract",
+        dataset_name="official_empty",
```

Comment:
"name" and "dataset_name" can be set as global variables because they are the same in every test